### PR TITLE
Microverse Trials Review

### PR DESCRIPTION
**Pull request by Jhaman1895**
Here we are reviewing our own code from one of our microverse trials projects.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <br>
+    <br>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+        <input 
+            type="email" 
+            id="email" 
+            placeholder="Enter Email here"
+            />
+        <input type="submit" id="submit">
+
```
Comment (Jhaman1895) [inline index.html:50]: I would suggest to remove all of the extra spaces in the markup.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+header{
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+}
+@media only screen and (max-width: 600px){
+    body { background-color: hotpink;
+
+    }
```
Comment (Jhaman1895) [inline style.css:10]: I suggest that you move the closing brace back to the same line as the "body" selector for cleaner code.

**Review by Jhaman1895 — COMMENTED** [review]
I suggest that you move the closing brace back to the same line as the "body" selector for cleaner code.

File: index.html (hunk 1 above)
Comment (HAMMAS-SALEEM) [reply index.html:50]: - [ ] It would be nice if you give a reason for the suggested change :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @Jhaman1895 :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Using GitHub Review
:heavy_check_mark: Using Inline comments

Good job on implementing the project requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
# Mandatory Changes

- [ ] you have given two reviews which is not a proper way for giving a review. It would be very nice if you give one review and add multiple inline comments. :blush:
- [ ] It would be very nice if you give reason for each requested change :+1: to let the student know why this change is important :+1:

# Optional suggestions
- [ ] Please use emojis :+1: as much as you can to make your review human-friendly :blush:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**